applications/notifications/server/notifications/services/notification_service.py

```diff
@@ -0,0 +1,28 @@
+from cloudharness import log as logger
+from cloudharness.utils.config import CloudharnessConfig as conf
+from notifications.services.notification.adapters import NotificationEmailAdapter
+from notifications.services.notification.backends import NotificationEmailBackend, NotificationConsoleBackend
+
+
+def send(operation, context):
+    notification_app = conf.get_application_by_filter(name='notifications')[0]
```

Comment:
still wondering why you don't want to use the cloudharness.application api, which is build upon the conf api but higher level.

Reply:
changed